faiss/impl/simd_result_handlers.h

```diff
@@ -217,6 +221,20 @@ struct ResultHandlerCompare : SIMDResultHandlerToFloat {
         return lt_mask;
     }
 
+    uint32_t get_lt_mask_for_range_search(size_t b) {
```

Comment:
Maybe this mask could take into account the sel, then we don't need separate conditions for with / without sel.

Reply:
A good suggestion. Let me see...